src/lib/components/InputChip/InputChip.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte/internal';
+
+	// Event Dispatcher
+	const dispatch = createEventDispatcher();
+
+	// Props (settings)
+	/** Optionally provide an semantic label. */
+	export let label: string = '';
+	/** Optionally provide an input placeholder. */
+	export let value: string[] = [];
+	/**
+	 * Provide a whitelist of accepted values.
+	 * @type {string[]}
+	 */
+	export let whitelist: string[] = [];
+	/** When enabled, will format entered values as lowercase. */
+	export let lowercase: boolean = true;
+	/**
+	 * Provide a custom validation function.
+	 * @type {function}
+	 */
+	export let validation: any = undefined;
```

Comment:
This could be typed as 

```ts
	/**
	 * Provide a custom validation function.
	 */
	export let validation: (...args: any[]) => boolean = () => true;
```

I would also suggest keeping the verb naming scheme for functions, so `isValid` rather than `validation`. You can leave it as `undefined` initialised, but we could also initialise it as a function that passes `true`- stylistic choice.